src/lib/base-pass.ts

```diff
@@ -71,6 +71,15 @@ export class PassBase extends PassStructure {
     else this.fields.serialNumber = v;
   }
 
+  get sharingProhibited() {
+    return this.fields.sharingProhibited;
+  }
+  
+  set sharingProhibited(v) {
+    if (!v) delete this.fields.sharingProhibited;
+    else this.fields.sharingProhibited = !!v;
```

Comment:
Will be just `else this.fields.sharingProhibited = true;` easier and better looking in this case? If this field cast to `false` we remove it, otherwise we set it to `true` - no need to double negate operation (that many companies considering antipattern in Typescript`